docs/data/base/components/modal/modal.md

```diff
@@ -15,7 +15,7 @@ waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/dialogmodal/
 Unstyled Modal is a utility component that renders its children in front of a backdrop.
 This lets you create an element that your users must interact with before continuing in the parent application.
 
-### Features
+**Features**:
```

Comment:
```suggestion
### Features
```
Would recommend reverting this, as this page structure is used in all base pages. We can discuss potential changes of the page structure in a dedicated PR.

Reply:
Oh, I actually started this PR by removing this H2 https://mui.com/base/react-modal/#customization and flatting the whole heading structure because it felt like it wasn't adding value. I mean, everything is about customization.